controllers/reivewsController.ts

```diff
@@ -0,0 +1,46 @@
+import { RequestHandler } from "express";
+
+import { PrismaClient, Prisma } from "@prisma/client";
+const prisma = new PrismaClient();
+
+export const approveReview: RequestHandler = async (req, res) => {
+    const reviewId: number = Number(req.params.reviewId);
```

Comment:
add type for req and res

Reply:
Types added. I was using the Request and Response types without importing them from express so it threw an error. Importing them from express fixed it.

 When I used the Request and Response types without importing them from express, was I accessing a global type Request and Response? It threwan error saying that params was not a property declared on the Request Type. Importing them from express fixed it.